pkg/setting/setting.go

```diff
@@ -177,6 +177,8 @@ var (
 
 	// Grafana.NET URL
 	GrafanaComUrl string
+	// Grafana.com API URL
+	GrafanaComAPIUrl string
```

Comment:
QQ: Is it possible for us to avoid this global variable?

Reply:
Yeah I think so, wasn't sure if they were still around for a reason so I just left them in. I'll get rid of it 👍

Reply:
Ah, just legacy, the `Cfg` struct is preferred 🙂